src/magql/filter.py

```diff
@@ -260,7 +246,7 @@ def generate_filters(table, info, *args, **kwargs):
                 )
                 filter_type = rel
             else:
-                magql_logger.error("Unknown field on sqlamodel")
+                raise KeyError(filter_name)
```

Comment:
Raises a `KeyError` now, but this should be re-examined later. It would already fail since `filter_type` would raise a `NameError` on the next line before.